src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,9 +33,21 @@
  */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private Analyze analyze = new Analyze();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
```

Comment:
Could you update the response with a sentiment score, for example? I know it won't be final, but this way we can see a reflection of the sentiment API into the response